rdr_service/alembic/versions/ec6018a5919f_modify_workspace_and_researcher_model.py

```diff
@@ -0,0 +1,130 @@
+    op.add_column('workbench_researcher', sa.Column('identifies_as_lgbtq', sa.Boolean(), nullable=True))
+    op.add_column('workbench_researcher', sa.Column('lgbtq_identity', sa.String(length=250), nullable=True))
+    op.drop_column('workbench_researcher', 'sexual_orientation')
+    op.add_column('workbench_researcher_history', sa.Column('degree', rdr_service.model.utils.Enum(WorkbenchResearcherDegree), nullable=True))
+    op.add_column('workbench_researcher_history', sa.Column('identifies_as_lgbtq', sa.Boolean(), nullable=True))
+    op.add_column('workbench_researcher_history', sa.Column('lgbtq_identity', sa.String(length=250), nullable=True))
+    op.drop_column('workbench_researcher_history', 'sexual_orientation')
+    op.add_column('workbench_workspace', sa.Column('access_to_care', rdr_service.model.utils.Enum(WorkbenchWorkspaceAccessToCare), nullable=True))
+    op.add_column('workbench_workspace', sa.Column('age', sa.JSON(), nullable=True))
+    op.add_column('workbench_workspace', sa.Column('disability_status', rdr_service.model.utils.Enum(WorkbenchWorkspaceDisabilityStatus), nullable=True))
+    op.add_column('workbench_workspace', sa.Column('education_level', rdr_service.model.utils.Enum(WorkbenchWorkspaceEducationLevel), nullable=True))
+    op.add_column('workbench_workspace', sa.Column('ethical_legal_social_implications', sa.Boolean(), nullable=True))
+    op.add_column('workbench_workspace', sa.Column('focus_on_underrepresented_populations', sa.Boolean(), nullable=True))
+    op.add_column('workbench_workspace', sa.Column('gender_identity', rdr_service.model.utils.Enum(WorkbenchWorkspaceGenderIdentity), nullable=True))
+    op.add_column('workbench_workspace', sa.Column('geography', rdr_service.model.utils.Enum(WorkbenchWorkspaceGeography), nullable=True))
+    op.add_column('workbench_workspace', sa.Column('income_level', rdr_service.model.utils.Enum(WorkbenchWorkspaceIncomeLevel), nullable=True))
+    op.add_column('workbench_workspace', sa.Column('others', sa.String(length=500), nullable=True))
+    op.add_column('workbench_workspace', sa.Column('race_ethnicity', sa.JSON(), nullable=True))
+    op.add_column('workbench_workspace', sa.Column('scientific_approaches', sa.String(length=500), nullable=True))
+    op.add_column('workbench_workspace', sa.Column('sex_at_birth', rdr_service.model.utils.Enum(WorkbenchWorkspaceSexAtBirth), nullable=True))
+    op.add_column('workbench_workspace', sa.Column('sexual_orientation', rdr_service.model.utils.Enum(WorkbenchWorkspaceSexualOrientation), nullable=True))
+    op.drop_column('workbench_workspace', 'reason_for_investigation')
+    op.add_column('workbench_workspace_history', sa.Column('access_to_care', rdr_service.model.utils.Enum(WorkbenchWorkspaceAccessToCare), nullable=True))
+    op.add_column('workbench_workspace_history', sa.Column('age', sa.JSON(), nullable=True))
```

Comment:
Age is set to a JSON type, is age going to be JSON data or just an integer value?  This applies to both 'workbench_workspace' and 'workspace_workspace_history' tables.

Reply:
It will be an array with integer values like `[1,3]`, the integer values are from the enum definition.

Reply:
Ok